apps/nextjs-website/src/app/auth/sign-up/page.tsx

```diff
@@ -35,11 +35,7 @@ const SignUp = () => {
   const [userAlreadyExist, setUserAlreadyExist] = useState(false);
 
   const goToConfirmSignUp = useCallback(() => {
-    router.replace(
-      `/auth/sign-up?email=${encodeURIComponent(userData.username)}&step=${
-        SignUpSteps.CONFIRM_SIGN_UP
-      }`
-    );
+    router.replace(`/auth/sign-up?step=${SignUpSteps.CONFIRM_SIGN_UP}`);
```

Comment:
Isn't it enough to `setSignUpStep(SignUpSteps.CONFIRM_SIGN_UP);`. Is the route persistent if the user refreshes the page?

Reply:
ok